src/main/java/com/codegym/jrugotom5/service/AdvertService.java

```diff
@@ -38,5 +41,20 @@ public List<AdvertBasicInfoDTO> getAllAdverts() {
         return Streamable.of(advertRepository.findAll())
                 .map(advert->modelMapper.map(advert, AdvertBasicInfoDTO.class))
                 .toList();
+      
+    public List<AdvertInfoForCreatorDto> getAdvertsByCreateBy(Long id) {
+        try{
+            List<Advert> adverts = advertRepository.getAdvertsByCreatedBy_Id(id);
+            if(adverts.isEmpty()){
```

Comment:
For what reason this check?
Repository method can return empty list if query found 0 records in DB.